src/ffmpeg-process.ts

```diff
@@ -39,6 +40,10 @@ export class FfmpegProcess {
       logInfo = logger?.info || noop,
       logPrefix = logLabel ? `${logLabel}: ` : ''
 
+    this.ff.stdout.on('data', (data: any) => {
+      options.stdoutCallback?.(data)
+    })
```

Comment:
Would you mind flipping the logic a bit here, only registering the `data` listener _if_ the `stdoutCallback` function is passed in? It's probably not a big deal, but it would be nice to avoid a bunch of noop function calls for the majority of cases where `stdoutCallback` isn't present.

Reply:
Agreed. Changes made.